main/dpkg/APKBUILD

```diff
@@ -1,17 +1,16 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dpkg
-pkgver=1.19.2
-pkgrel=1
+pkgver=1.19.4
+pkgrel=0
 pkgdesc="The Debian Package Manager"
 url="http://packages.debian.org/dpkg"
 arch="all"
-license="GPL"
-depends="xz"
+license="GPL-2.0-or-later BSD-2-Clause"
+depends="xz tar"
 options="!check"
-makedepends="bzip2-dev zlib-dev perl linux-headers"
+makedepends="bzip2-dev linux-headers perl zlib-dev"
 subpackages="$pkgname-doc $pkgname-dev"
 source="http://ftp.de.debian.org/debian/pool/main/d/$pkgname/${pkgname}_$pkgver.tar.xz"
-builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
if you remove this var; won't the 'cd $builddir' fail below in prepare?

Reply:
No, since "builddir="$srcdir"/$pkgname-$pkgver" is the default path. From AL3.9 forward it is acceptable to remove. I can not find the statement atm. I got it pointed out in https://github.com/alpinelinux/aports/pull/6275#discussion_r255364603

Thanks for the feedback. It made me have another look at the prepare function.

I turnouts that prepare was completely unnecessary since the dpkg team had made code changes to deal with cases where ncursesw is not used. Plus sed is problematic since it does not give a failed notification.